### PR TITLE
Update note section

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1791

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: docset/windows/nettcpip/Set-NetTCPSetting.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ To apply a TCP setting to a port number or destination IP address range, create
 > [!Note]
```
Comment (JohanFreelancer9) [inline docset/windows/nettcpip/Set-NetTCPSetting.md:61]:
```suggestion
> [!NOTE] 
```

File: docset/windows/nettcpip/Set-NetTCPSetting.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ To apply a TCP setting to a port number or destination IP address range, create
 > [!Note] 
 >1) You can modify Custom and Non-Custom settings on windows server 2016 and 2019.
```
Comment (JohanFreelancer9) [inline docset/windows/nettcpip/Set-NetTCPSetting.md:62]:
```suggestion
>1) You can modify custom and non-custom settings on Windows server 2016 and 2019.
```

File: docset/windows/nettcpip/Set-NetTCPSetting.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ To apply a TCP setting to a port number or destination IP address range, create
...
 >2) You can modify only Custom settings, Internet and Datacenter settings Cannot be modified on windows 2012 or earlier versions.
->3) You cannot modify the NetTCPsetting on Client Operating systems(Windows 7, 8.1 and 10) as they are Read-Only.
+>3) On client operating systems (Windows 7, 8.1 and 10), only AutoReusePortRangeStartPort and AutoReusePortRangeNumberOfPorts parameters could be modified. Others are Read-Only.
```
Comment (JohanFreelancer9) [inline docset/windows/nettcpip/Set-NetTCPSetting.md:64]:
```suggestion
>3) On client operating systems (Windows 7, 8.1 and 10), only AutoReusePortRangeStartPort and AutoReusePortRangeNumberOfPorts parameters can be modified. Others are read-only.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by e0i** [comment]
Re opening to run checks

**Comment by e0i** [comment]
Closing and re-opening to refresh validation checks.

File: docset/windows/nettcpip/Set-NetTCPSetting.md [hunk 4]
```diff
@@ -58,10 +58,10 @@ The **Set-NetTCPSetting** cmdlet modifies a TCP setting.
 TCP settings are optimized for different network conditions including latency and congestion.
 To apply a TCP setting to a port number or destination IP address range, create a transport filter by using the New-NetTransportFilter cmdlet.
 
-> [!Note] 
->1) You can modify Custom and Non-Custom settings on windows server 2016 and 2019.
+> [!NOTE] 
+>1) You can modify custom and non-custom settings on Windows server 2016 and 2019.
 >2) You can modify only Custom settings, Internet and Datacenter settings Cannot be modified on windows 2012 or earlier versions.
->3) You cannot modify the NetTCPsetting on Client Operating systems(Windows 7, 8.1 and 10) as they are Read-Only.
+>3) On client operating systems (Windows 7, 8.1 and 10), only AutoReusePortRangeStartPort and AutoReusePortRangeNumberOfPorts parameters can be modified. Others are read-only.
```
Comment (JohanFreelancer9) [inline docset/windows/nettcpip/Set-NetTCPSetting.md:64]:
```suggestion
>3) On client operating systems (Windows 7, 8.1, and 10), only AutoReusePortRangeStartPort and AutoReusePortRangeNumberOfPorts parameters can be modified. Others are read-only.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@MaratMussabekov - apologies, one more, please.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Review by get-itips — CHANGES_REQUESTED** [review]
Can you please replace the NOTE tag with a regular Note?

**Review by dariomws — CHANGES_REQUESTED** [review]
I tested it on Windows 10 and I could also update the following parameters:
- MemoryPressureProtection
- AutoTuningLevelLocal
- EcnCapability
- Timestamps
- InitialRtoMs
- ScalingHeuristics
- DynamicPortRangeStartPort
- DynamicPortRangeNumberOfPorts
- NonSackRttResiliency
- ForceWS
- MaxSynRetransmissions

Agree with @get-itips, but I think we need to test these parameters carefully on the different Win client OS.
And I think that there are more parameters we can modify, so I list of the read-only parameters could be a better idea.

File: docset/windows/nettcpip/Set-NetTCPSetting.md [hunk 5]
```diff
@@ -58,10 +58,10 @@ The **Set-NetTCPSetting** cmdlet modifies a TCP setting.
 TCP settings are optimized for different network conditions including latency and congestion.
 To apply a TCP setting to a port number or destination IP address range, create a transport filter by using the New-NetTransportFilter cmdlet.
 
-> [!Note] 
->1) You can modify Custom and Non-Custom settings on windows server 2016 and 2019.
+> [!NOTE]
```
Comment (JohanFreelancer9) [inline docset/windows/nettcpip/Set-NetTCPSetting.md:61]:
```suggestion
**Note:**
```

File: docset/windows/nettcpip/Set-NetTCPSetting.md [hunk 6]
```diff
@@ -58,10 +58,10 @@ The **Set-NetTCPSetting** cmdlet modifies a TCP setting.
 TCP settings are optimized for different network conditions including latency and congestion.
 To apply a TCP setting to a port number or destination IP address range, create a transport filter by using the New-NetTransportFilter cmdlet.
 
-> [!Note] 
->1) You can modify Custom and Non-Custom settings on windows server 2016 and 2019.
+> [!NOTE] 
+>1) You can modify custom and non-custom settings on Windows server 2016 and 2019.
 >2) You can modify only Custom settings, Internet and Datacenter settings Cannot be modified on windows 2012 or earlier versions.
```
Comment (JohanFreelancer9) [inline docset/windows/nettcpip/Set-NetTCPSetting.md:63]:
```suggestion
>2) You can modify only custom settings. Internet and Datacenter settings cannot be modified on Windows 2012 or earlier versions.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Removed the NOTE tag and added some additional edits.

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by MaratMussabekov** [comment]
Hello @dariomws ,
**I think that there are more parameters we can modify, so I list of the read-only parameters could be a better idea.** - done, please have a look
**I think we need to test these parameters carefully on the different Win client OS.** - tested for windows 10 (as the article is about Windows 10). For windows 8.1 and 7, the set of parameters is different, and they can be changed only for InternetCustom and DataCenterCustom custom settings. 

Thank you

File: docset/windows/nettcpip/Set-NetTCPSetting.md [hunk 7]
```diff
@@ -58,10 +58,17 @@ The **Set-NetTCPSetting** cmdlet modifies a TCP setting.
...
 To apply a TCP setting to a port number or destination IP address range, create a transport filter by using the New-NetTransportFilter cmdlet.
 
-> [!Note] 
->1) You can modify Custom and Non-Custom settings on windows server 2016 and 2019.
->2) You can modify only Custom settings, Internet and Datacenter settings Cannot be modified on windows 2012 or earlier versions.
->3) You cannot modify the NetTCPsetting on Client Operating systems(Windows 7, 8.1 and 10) as they are Read-Only.
+**Note:**
+>1) You can modify custom and non-custom settings on Windows server 2016 and 2019.
+>2) You can modify only custom settings. Internet and Datacenter settings cannot be modified on Windows 2012 or earlier versions.
+>3) On Windows 10, following parameters are read-only and cannot be modified:
```
Comment (JohanFreelancer9) [inline docset/windows/nettcpip/Set-NetTCPSetting.md:64]:
```suggestion
>3) On Windows 10, the following parameters are read-only and cannot be modified:
```

File: docset/windows/nettcpip/Set-NetTCPSetting.md [hunk 8]
```diff
@@ -58,10 +58,17 @@ The **Set-NetTCPSetting** cmdlet modifies a TCP setting.
...
+
```
Comment (JohanFreelancer9) [inline docset/windows/nettcpip/Set-NetTCPSetting.md:65]:
```suggestion
```

File: docset/windows/nettcpip/Set-NetTCPSetting.md [hunk 9]
```diff
@@ -58,10 +58,17 @@ The **Set-NetTCPSetting** cmdlet modifies a TCP setting.
...
+* MinRtoMs
```
Comment (JohanFreelancer9) [inline docset/windows/nettcpip/Set-NetTCPSetting.md:66]:
```suggestion
>* MinRtoMs
```

File: docset/windows/nettcpip/Set-NetTCPSetting.md [hunk 10]
```diff
@@ -58,10 +58,17 @@ The **Set-NetTCPSetting** cmdlet modifies a TCP setting.
...
+* InitialCongestionWindowMss
```
Comment (JohanFreelancer9) [inline docset/windows/nettcpip/Set-NetTCPSetting.md:67]:
```suggestion
>* InitialCongestionWindowMss
```

File: docset/windows/nettcpip/Set-NetTCPSetting.md [hunk 11]
```diff
@@ -58,10 +58,17 @@ The **Set-NetTCPSetting** cmdlet modifies a TCP setting.
...
+* AutomaticUseCustom
+* CongestionProvider
```
Comment (JohanFreelancer9) [inline docset/windows/nettcpip/Set-NetTCPSetting.md:69]:
```suggestion
>* CongestionProvider
```

File: docset/windows/nettcpip/Set-NetTCPSetting.md [hunk 12]
```diff
@@ -58,10 +58,17 @@ The **Set-NetTCPSetting** cmdlet modifies a TCP setting.
...
+* CwndRestart
```
Comment (JohanFreelancer9) [inline docset/windows/nettcpip/Set-NetTCPSetting.md:70]:
```suggestion
> * CwndRestart
```

File: docset/windows/nettcpip/Set-NetTCPSetting.md [hunk 13]
```diff
@@ -58,10 +58,17 @@ The **Set-NetTCPSetting** cmdlet modifies a TCP setting.
...
->2) You can modify only Custom settings, Internet and Datacenter settings Cannot be modified on windows 2012 or earlier versions.
->3) You cannot modify the NetTCPsetting on Client Operating systems(Windows 7, 8.1 and 10) as they are Read-Only.
+**Note:**
+>1) You can modify custom and non-custom settings on Windows server 2016 and 2019.
+>2) You can modify only custom settings. Internet and Datacenter settings cannot be modified on Windows 2012 or earlier versions.
+>3) On Windows 10, following parameters are read-only and cannot be modified:
+
+* MinRtoMs
+* InitialCongestionWindowMss
+* AutomaticUseCustom
```
Comment (JohanFreelancer9) [inline docset/windows/nettcpip/Set-NetTCPSetting.md:68]:
```suggestion
> * AutomaticUseCustom
```

File: docset/windows/nettcpip/Set-NetTCPSetting.md [hunk 14]
```diff
@@ -58,10 +58,17 @@ The **Set-NetTCPSetting** cmdlet modifies a TCP setting.
...
+* CongestionProvider
+* CwndRestart
+* DelayedAckTimeoutMs
```
Comment (JohanFreelancer9) [inline docset/windows/nettcpip/Set-NetTCPSetting.md:71]:
```suggestion
> * DelayedAckTimeoutMs
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by dariomws — APPROVED** [review]
I'm ok after you apply Johan suggestions.

**Comment by get-itips** [comment]
> I'm ok after you apply Johan suggestions.

Me too

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**